services/src/test/resources/mods/invalid-mods.xml

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Not so much invalid mods as invalid XML, but it will do :)

Reply:
well, it's empty xml, and hence invalid mods